include/circt/Dialect/LLHD/IR/MemoryOps.td

```diff
@@ -27,7 +27,7 @@ def LLHD_VarOp : LLHD_Op<"var", [
   }];
 
   let arguments = (ins LLHD_AnyUnderlyingType: $init);
-  let results = (outs LLHD_AnyPtrType: $result);
+  let results = (outs Res<LLHD_AnyPtrType, "The allocated reference.", [MemAlloc]>: $result);
```

Comment:
Looking at the description the other dialects provide here lets me suggest that it's designed to be part of a sentence instead of a whole sentence. Though, I'm not sure. Do you know what this description is used for?

Reply:
I thought this would replace the operand description for that specific operand in the generated doc, but it seems not to be the case. It doesn't seem to be used anywhere in the doc generated through `mlir-doc`...

Reply:
Yes, I also couldn't find any clue on what this is used for. Was just wondering if I overlooked something.